tests/testdata/multiple/a.ts

```diff
@@ -55,3 +55,11 @@ export function qaz(a: number);
 export function qaz(a: string | number) {}
 
 export const c = (): string => "hello";
+
+export function d(
+  foo: number = 1,
+  bar: string = "bar",
+  baz: { hello?: string } = {},
```

Comment:
This is another case that displays `[UNSUPPORTED]`. If this PR doesn't fix this case, I wouldn't consider it blocking. It can be fixed in a follow-up PR.
```suggestion
  baz: { hello?: string } = {},
  qux = bar.length,
```

Reply:
That case seems rendered like the below:

<img width="853" alt="Screenshot 2024-07-09 at 16 21 41" src="https://github.com/denoland/deno_doc/assets/613956/e3cf64a6-7c81-4779-a260-185fdf472876">

Reply:
Why are there square brackets?

Reply:
I'm not sure. cc @crowlKats